CHANGELOG.md

```diff
@@ -10,14 +10,20 @@
 
 ### Fixed
 
+## [0.3.0] - 2020-05-19
+
+### Changed
+
+- :fire: `Teal dark`: changed hex value from `#344b63` to `#2a3b4d`. ([@driesd](https://github.com/driesd) in [#3])
+
 ## [0.2.0] - 2020-03-31
 
 ### Changed
 
-- `Teal darkest`: changed hex value from `#2a3b4d` to `#1a1c20`. ([@driesd](https://github.com/driesd) in [#2](https://github.com/teamleadercrm/ui-colors/pull/2))
+- :fire: `Teal darkest`: changed hex value from `#2a3b4d` to `#1a1c20`. ([@driesd](https://github.com/driesd) in [#2])
```

Comment:
Why fire? 😄

Reply:
My intension was to indicate that this could be a breaking change (contrast wise), but there's probably a better gitmoji for that... 😇

Reply:
💥 is the one used for breaking changes 💥